rb_ws/src/buggy/scripts/visualization/telematics.py

```diff
@@ -14,8 +14,10 @@ class Telematics:
     def __init__(self):
         """Generate all the subscribers and publishers that need to be reformatted.
         """
-        self.odom_subscriber = rospy.Subscriber("/nav/odom", Odometry, self.convert_odometry_to_navsatfix)
-        self.odom_publisher = rospy.Publisher("/nav/odom_NavSatFix", NavSatFix, queue_size=10)
+
+        # TODO: add buggy name to argparse
```

Comment:
Why has this TODO not been finished?

Reply:
deprecated as from description of PR:

"NAND's position does need to be republished in order for it to be visualized"

so this is a matter of removing the todo since we only rebroadcast nand pos